Celeste.Mod.mm/Patches/DreamBlock.cs

```diff
@@ -187,3 +195,29 @@ public patch_DreamBlock(EntityData data, Vector2 offset)
         }
     }
 }
+
+namespace MonoMod {
+    /// <summary>
+    /// Patches <see cref="DreamBlock.Setup()" /> to fix issue #556.
```

Comment:
I think the description should actually tell you what the patch does without having to check a Github issue.

Reply:
Fair point, I was just copying the format from another patch class  that directly references GitHub.